src/main/java/org/jenkinsci/plugins/workflow/flow/FlowExecutionList.java

```diff
@@ -164,15 +159,11 @@ public static FlowExecutionList get() {
     }
 
     /**
-     * Returns true if all executions that were present in this {@link FlowExecutionList} have been loaded and resumed.
-     *
-     * This takes place slightly after {@link InitMilestone#COMPLETED} is reached during Jenkins startup.
-     *
-     * Useful to avoid resuming Pipelines in contexts that may lead to deadlock.
+     * @deprecated Only exists for binary compatibility.
      */
-    @Restricted(Beta.class)
+    @Deprecated
     public boolean isResumptionComplete() {
-        return resumptionComplete;
+        return false;
     }
```

Comment:
Maybe `true` would be more compatible for current versions of `workflow-durable-task-step`? Or is it the reverse?

Reply:
`false` prevents the code added in https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/188 (and https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/185) from ever taking effect, like if they were never merged and released in the first place.